contracts/interfaces/ISwapRouter__NoDeadline.sol

```diff
@@ -1,3 +1,7 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.2;
+
```

Comment:
why not use the specific `0.8.26` version?

Reply:
because since it's just an interface, it can be used with any version, like in the case we have an external contract we want to include in our tests that uses v0.6 or 0.7